HW_1/task_2.py

```diff
@@ -0,0 +1,31 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+s = int(input("Введите общее количество журавликов, которые сделали Петя, Катя и Сережа: "))
+
+# Вычисляем количество журавликов, которые сделала Катя
+katya = 2 * (s // 3)
+
+# Вычисляем количество журавликов, которые сделали Петя и Сережа
+petya_and_sereja = s // 3
+
+# Делим количество журавликов Пети и Сережи поровну
+petya = petya_and_sereja // 2
+sereja = petya_and_sereja // 2
```

Comment:
Это можно тоже совместить. И, Михаил, в Python, да и в других языках программирования не принято использовать в качестве имен переменных транслитерацию русских слов.
```suggestion
petya = sereja = petya_and_sereja // 2
```